internal/config/azure_keyvault.go

```diff
@@ -1,7 +1,57 @@
 package config
 
-import "fmt"
+import (
+	"context"
+	"fmt"
 
-func loadFromAzureKeyVault() (*Config, error) {
-	return nil, fmt.Errorf("loading config from Azure Key Vault is not implemented yet")
+	operatorcontext "altinn.studio/altinn-k8s-operator/internal/operator_context"
```

Comment:
Curious about why the module is called "altinn.studio/altinn-k8s-operator" instead of "github.com/altinn/altinn-k8s-operator"

Reply:
good point, thanks for reviewing: https://github.com/Altinn/altinn-k8s-operator/pull/14